static/app/views/acceptProjectTransfer/index.tsx

```diff
@@ -42,8 +42,13 @@ class AcceptProjectTransfer extends DeprecatedAsyncView<Props, State> {
       },
       success: () => {
         const orgSlug = formData.organization;
-
-        this.props.router.push(normalizeUrl(`/organizations/${orgSlug}/projects/`));
+        const projectSlug = this.state?.transferDetails?.project.slug;
+        const sentryUrl = ConfigStore.get('links').sentryUrl;
+        if (projectSlug === null) {
+          window.location.href = `${sentryUrl}/organizations/${orgSlug}/projects/`;
+        } else {
+          window.location.href = `${sentryUrl}/settings/${orgSlug}/projects/${projectSlug}/teams/`;
+        }
         addSuccessMessage(t('Project successfully transferred'));
```

Comment:
One small note here is that this success message won't be seen anymore since we are loading a new page. Hopefully being redirected to the new org/project is enough to show the user it worked? Because it would be a little more difficult to show the success message on the page you are being directed to

Reply:
Yeah I think just being in the project page on the new org should be enough indication